main.rb

```diff
@@ -1,15 +1,9 @@
-require './app'
+require_relative 'app'
 require_relative 'interface'
+require_relative 'data_store'
 
-class Main
-  def initialize
-    @app = App.new
-  end
-
-  def run
-    Interface.new(@app).start
-  end
+def main
+  checking_options(hello_message)
```

Comment:
- [ ] I believe this file is supposed to start the app but at the moment, it crashes when I try to run it so I can test other features if they are working as expected. Kindly work on the app so it runs successfully 👍🏽
- [ ] Please be sure everything works fine before you submit for a review, okay 👍🏽

<img width="1366" alt="Screenshot 2023-03-15 at 11 32 37 PM" src="https://user-images.githubusercontent.com/50941074/225467107-089f309c-3fae-4903-ad57-a1f82762a81a.png">